legacy-controller-docs/404.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Red Hat Ansible Automation Controller legacy documentation</title>
+    <meta name="description" content="Red Hat Ansible Automation Controller" />
+    <link
+      rel="canonical"
+      href="https://legacy-controller-docs.ansible.com/404.html"
+    />
+  </head>
+  <body>
+    <img src="static/images/cow.png" alt="cowsay404" />
+  </body>
```

Comment:
Much tho I enjoy the moo... since we have a specific 404 page here, maybe we should add a link to the platform page?